packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -518,7 +518,7 @@ function TokensPanel({
         </div>
         {errorMessage && <p className="text-xs text-red-400">{errorMessage}</p>}
       </form>
-      <div className="flex flex-grow flex-col overflow-auto rounded-md border border-gray-4 lg:shadow-[0px_4px_10px_rgba(120,120,120,0.25)]">
+      <div className="tokenSearchList flex flex-grow flex-col overflow-auto rounded-md border border-gray-4 lg:shadow-[0px_4px_10px_rgba(120,120,120,0.25)]">
```

Comment:
I think `tokenSearchList` class is only used as a cypress selector here. According to [Cypress](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) we should use `data-` over `class` selectors. 
 
 Should we change it to `data-` selector?